src/utils/activity/index.ts

```diff
@@ -93,6 +93,13 @@ export const mergeActivityItems = (
 			needsSorting = true;
 			break;
 		}
+		if (
+			mergedItems[i].timestamp === mergedItems[i - 1].timestamp &&
```

Comment:
```suggestion
		if (
		         i > 0 &&
			mergedItems[i].timestamp === mergedItems[i - 1].timestamp &&
```

May need to add `i > 0` to the beginning of the condition to prevent an out of bounds error.

Reply:
`i` starts with `1`, so everything is safe here